api-common/src/main/java/io/enmasse/api/auth/TokenReview.java

```diff
@@ -4,14 +4,23 @@
  */
 package io.enmasse.api.auth;
 
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
 public class TokenReview {
     private final String userName;
     private final String userId;
+    private final List<String> groups;
```

Comment:
Should this be Sets rather than Lists?  You've gone to the trouble to define equality, so I'd expect groups={A,B} and groups={B,A} to be equal.